run-tests.php

```diff
@@ -1091,7 +1091,9 @@ function find_files(string $dir, bool $is_ext_dir = false, bool $ignore = false)
         }
 
         // Otherwise we're only interested in *.phpt files.
-        if (substr($name, -5) == '.phpt') {
+        // (but not those starting with a dot, which are hidden on
+        // many platforms)
+        if (substr($name, -5) == '.phpt' && substr($name, 0, 1) !== '.' ) {
```

Comment:
style nit:
```suggestion
        if (substr($name, -5) == '.phpt' && substr($name, 0, 1) !== '.') {
```
I also wonder whether we should make this better readable by using `str_starts_width()` and `str_ends_with()`, or whether we do not yet want to require PHP 8 for the test runner (or do we already?)

Reply:
`gen_stubs` is documented to only require PHP7.1, but `run_tests` is generally executed with the version-under-test, so using PHP 8isms should be fine I think?

Reply:
Latest version uses str_starts_with and str_ends_with().

Reply:
> […], but run_tests is generally executed with the version-under-test, […]

I'm not sure about that. If you do `make test …` it is, but I often see `php run-tests.php …`, and some may want to use customized test runners from higher PHP branches. From a quick check with barlett/php-compatinfo, it looks like we're currently requiring PHP 7.4.0 only.

Reply:
Ok, switched it back to PHP 7.4.